src/app/modules/forms/ix-forms/validators/url-validation.service.ts

```diff
@@ -2,5 +2,5 @@ import { Injectable } from '@angular/core';
 
 @Injectable({ providedIn: 'root' })
 export class UrlValidationService {
-  urlRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$/;
+  urlRegex = /^(https?|ftp|mqtt|ws|wss):\/\/([a-zA-Z0-9.-]+)(:[0-9]+)?(\/.*)?$/;
```

Comment:
Can we update regexp to match `whatever://anything`?
This still won't allow for some URLs.

Reply:
> Can we update regexp to match `whatever://anything`? This still won't allow for some URLs.

I'll check. In this case its really wild card 😃

Reply:
I updated it + tests.